plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -35,6 +35,9 @@
 #include <sys/types.h>
 #include <sys/xattr.h>
 
+#define GNOME_DESKTOP_USE_UNSTABLE_API "unstable"
```

Comment:
There's just this one where I couldn't find the standard practice on how to define this, so I went with `unstable`